entrypoint.sh

```diff
@@ -133,16 +133,24 @@ nginxConfiguration() {
 }
 additionalPuppetConfiguration() {
     echo "Executing additional puppet configuration ..."
+    
     if [ "$QUEUE_WORKERS_MULTIPROCESS" == "True" ] || [ "$QUEUE_WORKERS_MULTIPROCESS" == "true" ]; then
         echo "Setting queue workers to run in multiprocess mode ..."
         crudini --set /etc/zulip/zulip.conf application_server queue_workers_multiprocess true
     elif [ "$QUEUE_WORKERS_MULTIPROCESS" == "False" ] || [ "$QUEUE_WORKERS_MULTIPROCESS" == "false" ]; then
         echo "Setting queue workers to run in multithreaded mode ..."
         crudini --set /etc/zulip/zulip.conf application_server queue_workers_multiprocess false
     else
-        echo "No additional puppet configuration executed."
-        return 0
+        echo "No additional puppet configuration executed for queue workers."
```

Comment:
The original code skipped the `zulip-puppet-apply -f` if there are no changes to be made. That should probably be preserved (e.g. using a flag variable).